### DISCUSSION TITLE
Consider creating a generic block filter for safelink_utama.js

### DISCUSSION THREAD

**Discussion by realodix** [realodix/AdBlockID#733] [Ideas]
Mempertimbangkan untuk membuat filter `/safelink_utama.$script,third-party` untuk memblokir file `safelink_utama.js` atau `safelink_utama.min.js`.

**Sample URL**
- https://rawcdn.githack.com/marsksatriagotham/wvw/2f0ffa8f809ee9f561434d487ac43247e05b90e8/safelink_utama.js (#240)
- https://github.com/alfirohman/tutorapk.com/blob/master/safelink_utama.js (#242)
- https://github.com/marsksatriagotham/wvw/blob/master/safelink_utama.js (#240)
- https://github.com/marsksatriagotham/safelinku/blob/master/safelink_utama.js (#240)

**Penjelasan Lebih Lanjut**
- Filter ini bisa mengatasi [[SafelinkScript] tokubatch.com (#234)](https://github.com/realodix/AdBlockID/issues/234)

**Comment by realodix** [comment]
Terdapat https://github.com/marsksatriagotham/benoja/blob/master/safelink_utama1.js

Dipertimbangkan juga `/safelink_utama$script,third-party` atau dibuat terpisah `/safelink_utama1.$script,third-party`

**Comment by GrennKren** [comment]
wildcard di rules bukannya bisa ya?
`/safelink_utama*.$script,third-party `

klo kubaca, wildcard ini bisa berupa string kosong jadi aman aja klo takutnya ga kebaca safelink_utama.js

**Comment by Recehan-Slayer** [comment]
https://cdn.rawgit.com/KompiAjaib/js/master/safelink_utama_filesafelk.js

**Comment by realodix** [comment]
> https://cdn.rawgit.com/KompiAjaib/js/master/safelink_utama_filesafelk.js

Thank you, harusnya ini sudah terblokir

**Comment by Recehan-Slayer** [comment]
iya mksudnya itu sbg sample aja
